tests/Doctrine/Migrations/Tests/Tools/Console/Command/MigrateCommandTest.php

```diff
@@ -500,14 +501,18 @@ protected function setUp(): void
 
     private function alterMetadataTable(): void
     {
-        $originalTable = $this->connection->getSchemaManager()
+        $schemaManager = $this->connection->getSchemaManager();
```

Comment:
Probably out of scope here, but `getSchemaManager()` is also deprecated.

Reply:
Good point, but I don't have the required energy for this right now :weary:  I also think it will be a lot easier in April when we drop support for DBAL 2.

As you see in this PR, having `phpstan-deprecation-rules` enabled makes this quite involving.